static/src/stylesheets/module/journalism/_media.scss

```diff
@@ -302,9 +302,19 @@
     border-radius: 30px;
     height: 34px;
 
-    .from-content-api.from-content-api {
+    .from-content-api.from-content-api & {
         border-bottom: 0;
-        color: #ffffff;
+        background: $highlight-main;
+        color: $brightness-7;
+
+        &:hover,
+        &:focus {
+            background: darken($highlight-main, 5);
+        }
+
+        .inline-icon {
+            fill: currentColor;
```

Comment:
And sorry I guess I missed this somewhere too, but where is `currentColor` defined?

Reply:
It's a CSS keyword [meaning](https://css-tricks.com/currentcolor/) the current `color` by looking at the element or its ancestors.

Reply:
ooh good to know. thanks